ui/src/app/edge/history/common/energy/flat/flat.ts

```diff
@@ -15,7 +15,7 @@ export class FlatComponent extends AbstractFlatWidget {
     protected autarchyValue: number | null;
     private static readonly EXCEL_TYPE = 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet;charset=UTF-8';
     private static readonly EXCEL_EXTENSION = '.xlsx';
-
+    protected readonly isMobile = this.service.isSmartphoneResolution;
```

Comment:
I always recommend using the same names (i.e. 'isMobile' or 'isSmartphoneResolution') in both places. I'd go for 'isSmartphoneResolution' because it is more expressive.

Reply:
ok